app/src/main/java/org/oppia/app/onboarding/OnboardingActivity.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.onboarding
+
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import org.oppia.app.profile.ProfileActivity
+import javax.inject.Inject
+
+/** An activity that shows a temporary Onboarding page until the app is onborarded then navigates to [ProfileActivity]. */
```

Comment:
Update this comment or the SplashActivity comment both are same.

Reply:
Done